cmd/lima-guestagent/daemon_linux.go

```diff
@@ -21,6 +23,9 @@ func newDaemonCommand() *cobra.Command {
 		RunE:  daemonAction,
 	}
 	daemonCommand.Flags().Duration("tick", 3*time.Second, "tick for polling events")
+	daemonCommand.Flags().Int("tcp-port", 0, "use tcp server instead a UNIX socket")
```

Comment:
Do we still need this?
For security reason I don't like listening on TCP. (Even though we don't leak critical info from the guest agent now)

Reply:
Not required, I'll remove it